extensions/adobe/experience/customerJourneyManagement/email-channel-context.schema.json

```diff
@@ -0,0 +1,58 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/experience/customerJourneyManagement/email-channel-context",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adobe CJM ExperienceEvent - MessageProfile Email Channel Details",
+  "description": "Email channel fields for the MessageProfile associated with the Adobe CJM Experience Event.",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+
+  "definitions": {
+    "emailChannelContext": {
+      "properties": {
+        "xdm:address": {
```

Comment:
Because this is not a core standard XDM mixin, these fields should not be in the XDM namespace either. This field should be named "https://ns.adobe.com/xdm/experience/customerJourneyManagement/address"

Reply:
Hi @cmathis:  If we rename the field "https://ns.adobe.com/xdm/experience/customerJourneyManagement/address", then while ingesting or accessing the data, do we need to use the complete path, or it get stripped of while accessing the data through xed format?

cc: @aashishjassal

Reply:
@cmathis : Can you explain when we have to use the xdm namespace and when to use the complete uri path for defining the properties in mixin.

There are existing mixin like https://github.com/adobe/xdm/blob/master/extensions/adobe/experience/campaign/feedbackevent.schema.json which have properties defined using:

1. xdm namespance
2. uri style namespace
3. No namespaces.

Reply:
Every field in XDM has a namespace. The URI style is the long form when you do not have a registered shorthand such as xdm, repo, adcloud, etc. There is no registered "customerJourneyManagement" short name so you would use the longer URI form which is fine as well. 

The path to this address field in AEP would become "_experience/customerJourneyManagmenet/address"

Reply:
@cmathis : So if we are using the uri style namespaces for every field, and have a case like "domain" is a sub property of a "address" field. Then how will the path be mapped when we fetch the xdm given both "domain" and "address" have the same namespace path.
Do we have to access as: _experience/customerJourneyManagmenet/address. _experience/customerJourneyManagmenet/domain, or it will be _experience/customerJourneyManagmenet/address.domain as domain is a sub property of a address with same prefix namespace.

Reply:
No the XED converter in AEP will recognize that the parent namespace is the same as the child namespace and will note require the duplication. It will be the second example - _experience/customerJourneyManagmenet/address.domain